.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
+  - "/bin/docker-image-tool.sh"
+  - "/bin/find-spark-home*"
```

Comment:
These two look like they can be removed as well.

Reply:
These have been removed. Thanks for catching them. I used the spark project's autolabeler as a template and accidentally left those in there in my rush to get some eyeballs on these, as I didn't feel particularly qualified to make a decision on how things should be labeled - just willing to add the autolabeling.